articles/cognitive-services/Face/face-how-to-install-containers.md

```diff
@@ -103,7 +103,7 @@ Use the [docker run](https://docs.docker.com/engine/reference/commandline/run/)
 * Exposes TCP port 5000 and allocates a pseudo-TTY for the container
 * Automatically removes the container after it exits
 
-```Docker
+```DockerFile
```

Comment:
note: this still probably isn't a correct language block because this is actually a block of commands